src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,59 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface MovieCardProps {
+  movie: Movie;
+}
+
+export const MovieCard: React.FC<MovieCardProps> = ({ movie }) => {
+  const {
+    description,
+    imdbUrl,
+    imgUrl,
+    title,
+  } = movie;
+
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={imgUrl}
+            alt="Film logo"
```

Comment:
Lets change it like this to be consistent  
```suggestion
            alt="Movie logo"
```